### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by myroslavkhovalko**

### CODE REVIEW

File: machine/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,34 @@ redirect_from:
...
+`google`
+* Add support for non-default service account
+
+`openstack`
+* Add flag for metadata
+
+`softlayer`
+* Don't set the request method again
+
 ## 0.16.0 (2018-11-08)
```
Comment (thaJeztah) [inline machine/release-notes.md:38]: Looks like 0.16.1 is missing as well, could you add that as well? https://github.com/docker/machine/releases/tag/v0.16.1

**Review by thaJeztah — COMMENTED** [review]
Thanks! Left a comment about one more entry missing 🤗

File: machine/release-notes.md (hunk 1 above)
Comment (usha-mandya) [reply machine/release-notes.md:38]: Thank you @thaJeztah. Will merge this PR and create a new PR for 0.16.1

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**